packages/flutter/lib/src/rendering/object.dart

```diff
@@ -2421,6 +2421,11 @@ abstract class RenderObject extends AbstractNode with DiagnosticableTreeMixin im
   /// semantics tree to implement implicit accessibility scrolling on iOS where
   /// the viewport scrolls implicitly when moving the accessibility focus from
   /// a the last visible node in the viewport to the first hidden one.
+  ///
+  /// See also:
+  ///
+  /// * [RenderViewportBase.cacheExtent], used by viewports to extend their
+  /// semantics clip beyond their approximate paint clip.
```

Comment:
nit: indent

Reply:
Just to be clear, should the indent on the second line match the position of the `[` on the first line, or be indented even further?

Reply:
it should match the position of `[`